feature/aft/aft_base/otg_tests/aft_base/README.md

```diff
@@ -27,55 +29,81 @@ Establish RSVP Sessions between ATE:port3 and SUT:port3.
 *   Configure explicit null and ipv6 tunneling.
 *   BGP advertised routes from ATE:port3 must be pointing to the 2 tunnels in the DUT.
 
-Verifications
+## Procedure
+
+*   Gnmi set with REPLACE option to push the configuration DUT.
+*   ATE configuration must be pushed.
 
-*   BGP routes advertised from ATE:port1,port2 must have 2 nexthops.
-*   IS-IS routes advertised from ATE:port3 must have one next hop.
-*   BGP routes advertised from ATE:port3 must have 2 next hops pointing to tunnels.
+## Verifications
 
+*   BGP 2000 routes advertised from ATE:port1,port2 must have 2 nexthops.
+*   IS-IS 2000 routes advertised from ATE:port3 must have one next hop.
+*   BGP 100 routes advertised from ATE:port3 must have 2 next hops pointing to tunnels.
+*   Use gnmi Subscribe with ON_CHANGE option to /network-instances/network-instance/afts.
+*   Verify afts prefix advertised by BGP,ISIS.
+*   Verify its next hop group, number of next hop and its interfaces.
+*   Verify the number of next hop is same as expected.
+*   Verify all other leaves mentioned in the path section.
 
 ## AFT-1.1.1: AFT Base Link Down scenario 1
 
+### Procedure
+
 Bring down the link between ATE:port2 and DUT:port2
```

Comment:
Specify the method you want to use for this.  Are you using the OTG api to shut down at the ATE or gnmi to disable the DUT port?  (I suggest using ATE, because the focus of this test is not to interact with the DUT interfaces model, but rather focus on the afts paths)

Reply:
changed, explained otg api